### PR TITLE
ERC-1271 : Set to Last Call

### PR DESCRIPTION
**Pull request by PhABC**

### CODE REVIEW

File: EIPS/eip-1271.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ eip: 1271
 title: Standard Signature Validation Method for Contracts
 author: Francisco Giordano (@frangio), Matt Condon (@shrugs), Philippe Castonguay (@PhABC), Amir Bandeali (@abandeali1), Jorge Izquierdo (@izqui), Bertrand Masius (@catageek)
 discussions-to: https://github.com/ethereum/EIPs/issues/1271
-status: Review
+status: Last Call
```
Comment (lightclient) [inline EIPS/eip-1271.md:6]:
```suggestion
status: Last Call
review-period-end: 2021-07-26
```

**Review by lightclient — CHANGES_REQUESTED** [review]
@PhABC hey, this looks good on my end. I can't suggest in the diff, but I noticed the section named `Implementation` should be `Reference Implementation` now per EIP-1. If you could update that quickly, I can get this merged ASAP. Thanks!

**Comment by PhABC** [comment]
@lightclient Renamed the section! Good catch

**Review by lightclient — APPROVED** [review]
Thanks, looks good to me now.

**Comment by eip-automerger** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):

 - Trying to change EIP 1271 state from Review to Last Call

**Review by ItsYoutacRRS — COMMENTED** [review]
Please appr

**Pull request merged**